src/explore-education-statistics-frontend/src/components/Link.tsx

```diff
@@ -31,7 +31,11 @@ const Link = ({
 }: LinkProps) => {
   const handleAnalytics = () => {
     if (analytics) {
-      logEvent(analytics.category, analytics.action, window.location.pathname);
+      logEvent(
+        analytics.category,
+        analytics.action,
+        analytics.label ? analytics.label : window.location.pathname,
```

Comment:
Isn't this the location of the browser window not the link?

Reply:
The label is the link. If there is no link it replaces the link with the location of the browser window

Reply:
FYI I think you can short-hand this as `analytics.label || window.location.pathname`, for next time